src/components/nested-donut-chart/nested-donut-chart.component.less

```diff
@@ -0,0 +1,33 @@
+ux-nested-donut-chart {
+    display: inline-block;
+    position: relative;
+    width: 165px;
+    height: 165px;
```

Comment:
This width and height should be part of the example css. By default, the component ought to fit the size of its parent container.